bankroll/converter/modelconverter.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Callable, cast, List, Optional, Sequence, Tuple, Union
+
+try:
+    import pandas as pd
+except ImportError:
+    pass
+
+from bankroll.model import Position, Trade
+
+ConvertibleModel = Union[Position, Trade]
```

Comment:
This is a good way of doing it! It allows collections of heterogeneous values, though, as you've seen (and guarded against).

You could instead use a `TypeVar` to statically enforce that all of the values are the same type:

```suggestion
_ConvertibleModel = TypeVar('_ConvertibleModel', Position, Trade)
```

(sidenote: please prefix all private symbols with `_`)